spin_rate:  50 # Hertz
jacobian_frame_name: base_link  # MoveIt! planning frame. Often the last link in your tf tree
force_torque_frame_name: base # The frame of incoming force/torque data
force_torque_topic: wrench # The topic of incoming force/torque data
move_group_name: manipulator # The MoveGroup name of your robot
max_allowable_cmd_magnitude: 0.5 # The largest joint speed correction that can be published [rad/s]
outgoing_joint_vel_topic: /compliance_controller/compliance_velocity_adjustment # Joint delta-velocities
joint_limit_margin: 0.1 # added as a buffer to joint limits [radians]. If moving quickly, make this larger.
compliance_library:
  # Key equation: compliance_velocity[i] = wrench[i]/stiffness[i]
  stiffness: [2000, 2000, 2000, 100, 100, 100] # First three element are Newtons/(m/s), last 3 are N*m/(rad/s). Should be positive.
  damping: [10000, 10000, 10000, 10000, 10000, 10000] # First three element are Newtons/m, last 3 are N*m/rad. Should be positive.
  low_pass_filter_param: 10 # Larger > more smoothing on compliance calcs, but more lag
  deadband: [1, 1, 1, 1, 1, 1] # No reaction until force exceeds X [N, N, N, Nm, Nm, Nm]
  end_condition_wrench: [80, 80, 80, 60, 60, 60] # Safety feature. Halt if force exceeds [N, N, N, Nm, Nm, Nm]
  highest_allowable_force: 88 # Safety feature. Halt if force magnitude exceeds this across all dimensions [N]
  highest_allowable_torque: 50 # Safety feature. Halt if torque magnitude exceeds this across all dimensions [Nm]
